src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    public abstract void draw();

```

Comment:
```suggestion
    void draw();
``` 
this method  public abstract by default